include/engine/datafacade/internal_datafacade.hpp

```diff
@@ -911,6 +919,15 @@ class InternalDataFacade final : public BaseDataFacade
         return m_entry_class_id_list.at(eid);
     }
 
+    util::guidance::TurnBearing PreTurnBearing(const EdgeID eid) const override final
+    {
+        return m_pre_turn_bearing.at(eid);
+    }
+    util::guidance::TurnBearing PostTurnBearing(const EdgeID eid) const override final
+    {
+        return m_post_turn_bearing.at(eid);
```

Comment:
From chat: we are going to leave it this way, since it is consistent with other access patterns.